src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { Children, FC } from 'react';
+
+// Styles:
+import styles from './avatar.module.scss';
+import { AvatarProps, AvatarFallbackProps, AvatarIconProps } from './';
+import { mergeClasses } from '../../shared/utilities';
+import { Icon } from '../Icon';
+
+export const AVATAR_CLASS_SET = [
+    styles.disruptive,
+    styles.grey,
+    styles.blue,
+    styles.orange,
+    styles.green,
+    styles.violet,
+    styles.yellow,
+    styles.bluegreen,
+];
+
+const AvatarFallback: FC<AvatarFallbackProps> = ({
+    children,
+    classNames,
+    style = {},
+}) => {
+    const avatarClasses: string = mergeClasses([
+        styles.wrapperStyle,
+        classNames,
+        AVATAR_CLASS_SET[
+            Math.floor(Math.random() * 100) % AVATAR_CLASS_SET.length
+        ],
+    ]);
+
+    return (
+        <div className={avatarClasses} style={style}>
+            <span>{children}</span>
```

Comment:
```suggestion
            {children}
```
Is the span necessary here ?